spec.md

```diff
@@ -1005,19 +1083,19 @@ services do not need to implement the endpoint for bind requests.
 
 Credentials are a set of information used by an application or a user to
 utilize the service instance. If the Service Broker supports generation of
-credentials it MUST return `credentials` in the response for a request to
-create a Service Binding. Credentials SHOULD be unique whenever possible, so
-access can be revoked for each binding without affecting consumers of other
-bindings for the service instance.
+credentials it MUST return `credentials` in the response for a synchronous
```

Comment:
This could be interpreted as only seeing the creds for sync, not async. I think its ok to keep it generic:
```
If the Service Broker supports generation of credentials it MUST return
credentials in the response to create a Service Binding. 
```
And people will know that in the async case the "real response" flows later - that's the point of async.

Reply:
Changed!